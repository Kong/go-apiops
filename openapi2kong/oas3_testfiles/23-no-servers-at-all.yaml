openapi: 3.0.0
info:
  title: Simple API
  version: 1.0.0
paths:
  /hello:
    get:
      summary: Say Hello
      responses:
        '200':
          description: Successful response
          content:
            application/json:
              schema:
                type: object
                properties:
                  message:
                    type: string
                    example: Hello, world!