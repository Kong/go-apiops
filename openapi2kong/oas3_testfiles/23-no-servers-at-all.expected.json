{
  "_format_version": "3.0",
  "services": [
    {
      "host": "localhost",
      "id": "a49d53c1-c2ce-5d7c-a72c-e2c376de9e93",
      "name": "simple-api",
      "path": "/",
      "plugins": [],
      "port": 443,
      "protocol": "https",
      "routes": [
        {
          "id": "046fbd3e-5b29-5d21-b7dd-637c5b1060df",
          "methods": [
            "GET"
          ],
          "name": "simple-api_hello_get",
          "paths": [
            "~/hello$"
          ],
          "plugins": [],
          "regex_priority": 200,
          "strip_path": false,
          "tags": [
            "OAS3_import",
            "OAS3file_23-no-servers-at-all.yaml"
          ]
        }
      ],
      "tags": [
        "OAS3_import",
        "OAS3file_23-no-servers-at-all.yaml"
      ]
    }
  ],
  "upstreams": []
}